src/main/resources/static/js/subscription.js

```diff
@@ -497,10 +497,11 @@ jQuery(document).ready(function () {
     };
```

Comment:
I don't know if this changed since before or if it was the previous behavior... Icons show up to the left in the datatable when having a smaller screen (not mobile view, just slightly narrower than full width). A green + symbol and when you click it you expand and view everything that can't fit in the full table row. Then a red - symbol is displayed to minimize...  Those symbols are not centered in the column. Also remove the shade style on them, so they will be aligned with the rest of the buttons on the page.

Reply:
It's always been off center on some browsers. It's even like that in the official examples of responsive datatables. It does have a text align rule but it's not working great. When you say shade do you mean the circular shadow around the it?

Reply:
Ok, then it's nothing new at least. I'm testing in firefox btw. Yes the circular shadow around it :)

Reply:
I removed the box shadow and also the border since in explorer and edge it doesn't render the border correctly and leaves some circular pixels...